src/app/interfaces.ts

```diff
@@ -162,6 +162,52 @@ export interface IUserTagCategory {
     tags: IUserTag[];
 }
 
+export interface IComplaintTag {
+    id: string;
+    tag_uri: string;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'tag_uri' is not in camel case.](https://app.codacy.com/gh/wncc/instiapp-angular/pullRequest?prid=5455571)